src/workerd/api/actor-state.c++

```diff
@@ -919,16 +919,20 @@ v8::Local<v8::Value> deserializeV8Value(
     // counting it against our internal storage error metrics but also throw a KJ exception rather
     // than a jsExceptionThrown error to avoid confusing the normal termination handling code.
     // We don't expect users to ever actually see this error.
-    JSG_REQUIRE(!tryCatch.HasTerminated(),
-        Error, "isolate terminated while deserializing value from Durable Object storage; "
-        "contact us if you're wondering why you're seeing this");
+    auto jsException = tryCatch.Exception();
```

Comment:
suggestion: While you're here, mind scrubbing mentions of "actor storage" or "durable object storage"? We're going to want to use this function outside of the storage api soon. :smile:

Reply:
There are situations where `tryCatch.HasCaught()` is true but `tryCatch.Exception()` returns an empty `v8::Local`. In such cases, we want to treat it the same as `tryCatch.CanContinue() == false`